examples/robot_brain_cmaes_database/evaluator.py

```diff
@@ -7,7 +7,7 @@
 from revolve2.ci_group import fitness_functions, terrains
 from revolve2.ci_group.simulation import make_standard_batch_parameters
 from revolve2.modular_robot import ModularRobot
-from revolve2.modular_robot.body import ActiveHinge, Body
+from revolve2.modular_robot.body._common import ActiveHinge, Body
```

Comment:
if the classes from `_common` are supposed to be public, they should either be exported explicitly in the `body` module, or the underscore should be removed from `_common`, making it a public module.